src/selectors/profile.js

```diff
@@ -79,6 +79,23 @@ export const getCommittedRange: Selector<StartEndRange> = createSelector(
   }
 );
 
+export const getPreviewSelection: Selector<PreviewSelection> = state =>
+  getProfileViewOptions(state).previewSelection;
+
+export const getPreviewSelectionRange: Selector<StartEndRange> = createSelector(
```

Comment:
There should probably be a simple test exercising this selector, and hitting both branches.